src/components/vm/disks/diskAdd.jsx

```diff
@@ -397,7 +397,7 @@ export const AddDiskModalBody = ({ disk, idPrefix, isMediaInsertion, vm, vms, su
 
     const getPoolFormatAndDevice = (pool, volName) => {
         const params = { format: getDefaultVolumeFormat(pool), device: "disk" };
-        if (['dir', 'fs', 'netfs', 'gluster', 'vstorage'].indexOf(pool.type) > -1) {
+        if (volName && ['dir', 'fs', 'netfs', 'gluster', 'vstorage'].indexOf(pool.type) > -1) {
```

Comment:
To make sure I understand: Is this functionally required to fix this bug, or just an unrelated optimization? I.e. are there pool volumes with undefined names which would unintentionally match here?